rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,432 @@
+```
+- [2125597](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2125597?lang=eng&suite=cobalt&marcData=Y)
+  490 with ISSN + 830 and 773. There are 1037 bibs that have the same ISSN in a series statement but
+  with different volume subfield.
+```yaml
+"partOf": [
+  {   // 490 +830 contain duplicated info
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "1064-3745",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Methods in Molecular Biology, Methods and Protocols",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   // 773 
+    "title": "Springer eBooks",
+    "type": "Work"
+  }
+]
+```
+- [3001878](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3001878?lang=eng&suite=cobalt&marcData=Y)
+  Only 830 with no 490/440 or 773
+```yaml
+"partOf": [
+    {
+        "title": "Adamson Collection Wellcome Library", // subfield $a concatenated with $p
+        "type": "Series"
+    }
+]
+```
+- [1110225](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1110225?lang=eng&suite=cobalt&marcData=Y)
+Series statement in 440 with only the title
+```yaml
+"partOf": [
+    {
+        "title": "Monographs on inorganic and physical chemistry",
+        "type": "Series"
+    }
+]
+```
+- [1204561](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1204561?lang=eng&suite=cobalt&marcData=Y) 440 with id
+```yaml
+"partOf": [
+  {
+    "title": "Brill's studies in intellectual history; v. 13",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"             
+    },
+    "partOf": [
+      {
+        "id": "abcdefgh",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "0920-8607",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Brill's studies in intellectual history",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  }
+]
+```
+- [2301867](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2301867?lang=eng&suite=cobalt&marcData=Y) 
+  773 not overlapping with series and no id. 
+```yaml
+"partOf": [
+    {
+        "title": "Eighteenth Century collections online",
+        "type": "Work"
+    }
+]
+```
+- [1186777](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1186777?lang=eng&suite=cobalt&marcData=Y) 
+  773 with an id in subfield $w which links to another Work in our library (see example below) 
+```yaml
+"partOf": [
+   {
+    "title": "Basil Hood. Photograph album; page 9",   
+    "partOf": [
+      {
+        "id": "abcdefgh",
+        "identifiers": [
+          {
+            // identifier from $w
```

Comment:
Correct thing to do here would be to use that id to look up the existing work (which I think we will be added when we sort the shelfmarks that should really be ids), then denormalise the existing work into here with the relation embedder.

Reply:
yup, I think that makes a lot of sense

Reply:
For now, as shelfmarks are not ids, I think it would make sense to not include the identifers field and possibly the whole nested `partOf`

Reply:
@alexwlchan has started to pick this up and so it's likely that, given we're going to the TEI work first, that they will be by the time we implement library linking.

Reply:
\*swears loudly*

I haven't read this whole RFC, but this might get a bit complicated with shelfmarks. In particular:

* IDs in that field don't have the (Wcat) prefix
* Not every (Wcat)-prefixed ID in a 773-field appears in the shelfmarks (e.g. 570997i), but some of them do (e.g. 5894i)
* How do we tell these apart from an Iconographic Collections identifier?

aaaarrggghhh